frontend/src/services/api.ts

```
@@ -0,0 +1,84 @@
+        (acc as Record<string, unknown>)[camelKey] = toCamelCaseKeys((obj as Record<string, unknown>)[key]);
+        return acc;
+      },
+      {} as CamelCaseKeys<T>,
+    );
+  }
+  return obj as CamelCaseKeys<T>;
+}
+
+const axiosInstance = axios.create({
+  baseURL: `${API_URL}/api/v1/`,
+  timeout: import.meta.env.VITE_API_TIMEOUT ? parseInt(import.meta.env.VITE_API_TIMEOUT) : 10_000,
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)